lib/cc/cli/version_checker.rb

```diff
@@ -42,7 +42,7 @@ def latest_version
       end
 
       def print_new_version_message
-        warn "A new version (v#{latest_version}) is available"
+        warn "A new version (v#{latest_version}) is available. Upgrade installations are available at: https://github.com/codeclimate/codeclimate#packages"
```

Comment:
@pbrisbin Yuuup. Updated. I think I must have heard someone say the word "installation" while I was typing that.